metadata/metadata_test.go

```diff
@@ -148,26 +149,26 @@ var _ = Describe("Metadata", func() {
 	})
 
 	Context("AddLabels function", func() {
-		When("called with a pipelineRun containing nil Labels", func() {
-			pipelineRun := &v1beta1.PipelineRun{
+		When("called with an object containing nil Labels", func() {
+			pod := &corev1.Pod{
```

Comment:
why are using `pod` here?
is it just to simplify the test case involving `GetLabelsWithPrefix`?

Reply:
We were using `Release` or `Pipeline` resources for it. The problem is that  by moving the labels to `metadata` package, when importing the `metadata`it  causes a cycle import error. Using the Pod instead, we avoid the error and keep the test the way it was initially written.

Reply:
And, if I am not wrong, the tests are just about adding labels and annotations, so the object itself does not matter. Pod is the most basic object we can use for tests AIUI. +1 for using pod